README.md

```diff
@@ -40,7 +40,11 @@ The OQS project is supported by the [Post-Quantum Cryptography Alliance](https:/
 
 Details on each supported algorithm can be found in the [docs/algorithms](https://github.com/open-quantum-safe/liboqs/tree/main/docs/algorithms) folder.
 
-The list below indicates all algorithms supported by liboqs, but not all those algorithms have been selected for standardization by NIST, specifically Kyber (excluding the "-90s" variants), Dilithium (excluding the "-AES" variants), Falcon, and SPHINCS+ (excluding the "robust" variants). Activating only those standardized algorithms for use in `liboqs` can be facilitated by setting the [OQS_ALGS_ENABLED](CONFIGURE.md#oqs_algs_enabled) build configuration variable to `STD`. By default `liboqs` is built supporting all, incl. experimental, PQ algorithms listed below.
+The list below indicates all algorithms currently supported by liboqs, including experimental algorithms and already excluding algorithm variants pruned during the NIST competition, such as Kyber-90s or Dilithium-AES.
+
+The only algorithms currently selected for standardization are the "ML-KEM" and "ML-DSA" variants with their respective different bit strengths. `liboqs` will retain these algorithm names selected by NIST throughout the finishing stages of the standardization process, so users can rely on their presence going forward. If NIST changes the implementation details of these algorithms, `liboqs` will adapt the implementation such as for users to not be impacted by such potential change. For users interested in explicitly selecting the current "proposed draft standard" code, the variants with the suffix "-ipd" are made available. At this stage, "ml-kem-ipd" and "ml-kem" as well as "ml-dsa-ipd" and "ml-dsa" are functionally equivalent, denoted by the "alias" moniker below.
```

Comment:
The first sentence in this paragraph is a bit confusing to me. Falcon and SPHINCS+ have also been selected by NIST for standardization.

Reply:
> Falcon and SPHINCS+ have also been selected by NIST for standardization.

OK, I stand corrected -- I thought those names and parameter sets are not yet final. Is there a doc like FIPS 203 to point to?

Reply:
Changed as suggested. Please re-check and approve @dstebila to close the last issue out for 0.10.0.

Reply:
Falcon and SPHINCS+ have both been selected for standardization, but those names will change. For Falcon, there is no FIPS draft yet. For SPHINCS+, [there is](https://csrc.nist.gov/pubs/fips/205/ipd), but we don't yet have an implementation of "SLH-DSA" (and indeed I'm not sure what tweaks the draft makes to the Round 3 version of SPHINCS+).

I think it makes sense to reword this to state something like
```
The only algorithms in `liboqs` that implement NIST standards drafts are the "ML-KEM" and "ML-DSA" variants...
```
and additionally add a sentence saying
```
Falcon and SPHINCS+ have also been selected for standardization, but the `liboqs` implementations of these algorithms are currently tracking Round 3 submissions and not NIST standards drafts.
```
I think this conveys the commitment to keep the ML-KEM and ML-DSA names (and currently only those names) stable throughout the standardization process, while recognizing that Falcon and SPHINCS+ are also soon to be standardized. Whenever we have implementations of SLH-DSA and NTRU-DSA (or whatever other name NIST settles on for Falcon) we can add them to this list of stables names too.

Sorry, I should have caught this on my first pass.

Reply:
That sounds great Spencer, go for it.

Reply:
Marking this conversation as resolved, as this change has now been applied.